audio/monkeys-audio/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            fernandotcl monkeys-audio 3.99.6 release-
+revision                0
+categories              audio
+maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
+license                 Restrictive
+# Re license see: https://lists.debian.org/debian-legal/2007/09/msg00079.html
+description             Lossless audio compressor
+long_description        Monkeyʼs Audio Codec is a lossless compressor for audio files. \
+                        Files in this format can typically be recognized by the .ape extension.
+homepage                https://monkeysaudio.com
```

Comment:
This web page describes version 10.17 of a Windows program but the GitHub repository's latest version is 3.99.6. The GitHub repo describes itself as a "Port of Monkey's Audio Codec to Unix-like systems". Maybe it's best to remove this homepage and let users go to the GitHub repo first, so there's no ambiguity or confusion about the source of the software.

```suggestion
```